### ISSUE TITLE
Verify the Bruno Flatpak on Flathub

### ISSUE THREAD

**Issue by AligningEntropy** [usebruno/bruno#2589]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

Flatpak support was enabled in #623 and has (at the time of writing this) around 11,000 installs; however, the Flatpak currently is marked as unverified. 

This is problematic as some Linux distributions, such as [Mint](https://www.phoronix.com/news/Linux-Mint-Unverified-Flatpaks), are starting to hide unverified flatpaks by default (making them not show up in search) for security reasons and many users can be scared off from downloading something with these intimidating warning banners. 

[Per Flathub's documentation](https://docs.flathub.org/docs/for-app-authors/verification/), the fix for this requires @helloanoop to "verify" themselves by linking their Flatpak account to this repo / the usebruno website. The steps listed appear pretty straight forward, so hopefully this won't take much effort. 

### Mockups or Images of the feature

![Flathub Warning Banner](https://github.com/usebruno/bruno/assets/36208309/a55e0f4f-5dd3-4659-8093-f2c98d8d0fa0)

**Comment by AligningEntropy** [comment]
This is somewhat related to #2112 as there were some [comments](https://github.com/usebruno/bruno/issues/2112#issuecomment-2065109820) discussing this issue; however, because that issue topic was not specifically related to this verification component, I opened this issue for easier addressability.

**Comment by mirai-toto** [comment]
I would also like to add that in the bruno download webpage https://www.usebruno.com/downloads Flatpak is missing

**Comment by ulph0** [comment]
i just installed it by flatpak because the actual snap version doesn't work. wondering why this ticket is open since july, https://flathub.org/apps/com.usebruno.Bruno is the official flathub package, isn't it?

**Comment by kolappannathan** [comment]
+1

**Comment by helloanoop** [comment]
https://flathub.org/apps/com.usebruno.Bruno is the official distribution of bruno on flatpak.

We’ll be verifying the Flatpak account soon. @sreelakshmi-bruno @sanjai0py, could you please update the website to include the Flatpak installation option?

**Comment by JohnMastodon** [comment]
The flathub website also indicates the package has proprietary source code, does the verification will update this to show the MIT license or should another issue be created to address it?

**Comment by Its-treason** [comment]
> The flathub website also indicates the package has proprietary source code

This is correct. The version distributed via Flathub, GitHub Realeaes etc. includes proprietary code. The Bruno Team has an internal Fork, that includes Golden edition features.

**Comment by helloanoop** [comment]
We have verified Bruno on flatpak

<img width="929" height="426" alt="Image" src="https://github.com/user-attachments/assets/2b7c72b4-4a9d-49b2-9faa-09faa5cf5950" />

**Comment by dphaldes** [comment]
Sorry for necroposting but I would prefer if the flathub edition was the FOSS version since the store is a premier linux and foss store. You could also create 2 versions of the app there, one foss and another non-foss with more features. 

> This is correct. The version distributed via Flathub, GitHub Realeaes etc. includes proprietary code. The Bruno Team has an internal Fork, that includes Golden edition features.
